src/operators/test_identity.py

```diff
@@ -5,31 +5,50 @@
 Identity operators are used to compare the objects, not if they are equal, but if they are actually
 the same object, with the same memory location.
 """
-
+import copy
 
 def test_identity_operators():
     """Identity operators"""
 
     # Let's illustrate identity operators based on the following lists.
     first_fruits_list = ["apple", "banana"]
     second_fruits_list = ["apple", "banana"]
-    third_fruits_list = first_fruits_list
+    third_fruits_list = first_fruits_list   #Shallow copy
 
     # is
     # Returns true if both variables are the same object.
 
     # Example:
     # first_fruits_list and third_fruits_list are the same objects.
     assert first_fruits_list is third_fruits_list
+    assert first_fruits_list == third_fruits_list
+
+
+    third_fruits_list[0] = "Mango"
+    print(first_fruits_list)
+
+    assert first_fruits_list is third_fruits_list
+    assert first_fruits_list == third_fruits_list
+
+
+    third_fruits_list = copy.deepcopy(first_fruits_list)   #Deep copy copy
+    # third_fruits_list[0] = "Mango"
+    print(first_fruits_list)
+
+    assert first_fruits_list is not third_fruits_list
+    assert first_fruits_list == third_fruits_list
 
     # is not
     # Returns true if both variables are not the same object.
 
     # Example:
     # first_fruits_list and second_fruits_list are not the same objects, even if they have
     # the same content
-    assert first_fruits_list is not second_fruits_list
+    # assertsert first_fruits_list is not second_fruits_list
```

Comment:
Why did we comment this?

Reply:
I created a deep copy for practice. In this scenario, this assert was creating an error. So I did comment this line. Yes It's my mistake either I should have corrected it or removed it. Now I have corrected it.